src/react/deckgl.js

```diff
@@ -118,141 +129,19 @@ export default class DeckGL extends React.Component {
     this.props.onWebGLInitialized(gl);
 
     // Note: avoid React setState due GL animation loop / setState timing issue
-    this.layerManager = new LayerManager({gl});
+    this.layerManager = new LayerManager({gl, canvas});
```

Comment:
There's no change here in the way we're using `canvas`, just passing it along to `LayerManager` for `EventManager` event registration. Is there some caveat we're not covering here (hence the link)?